### DISCUSSION TITLE
Quadlet: Handle multiple Exec= entries

### DISCUSSION THREAD

**Discussion by wnhrt** [containers/podman#24948] [General]
### Feature request description

Allowing multiple `Exec=` entries, like with `Label=`, would provide better readability and maintainability.

A prime example is [the example docker-compose.yml of treafik](https://doc.traefik.io/traefik/user-guides/docker-compose/basic-example/) with four long entries.
My own configuration is 18 arguments long and at exactly 1000 characters not easily manageable in a single line.

### Suggest potential solution

Allowing multiple `Exec=` entries, like with `Label=`.

### Have you considered any alternatives?

No. I am in the progress of migrating from docker compose and have not much experience with quadlets.
If there is an alternative approach, which I overlooked, please let me know.

### Additional context

Related issue: https://github.com/containers/podman/issues/22027

I was able to start a container with multiple `Exec=` entries but `podman ps` shows only the arguments from the last entry.
This case, silently ignoring present configuration, seems more problematic than failing with an error.

```
[Container]
Image=docker.io/library/traefik
Exec="--api.insecure=true"
Exec="--providers.docker=true"
Exec="--providers.docker.exposedbydefault=false"
Exec="--entryPoints.web.address=:80"
Notify=true
Volume=%t/podman/podman.sock:/var/run/docker.sock
SecurityLabelDisable=true
```

```
CONTAINER ID  IMAGE                             COMMAND               CREATED        STATUS        PORTS       NAMES
1ab2d3e4f5g6  docker.io/library/traefik:latest  --entryPoints.web...  7 minutes ago  Up 7 minutes  80/tcp      systemd-traefik
```

**Comment by ygalblum** [comment]
When deciding whether to all the occurrences of a key or only the last, Quadlet follows what is supported by podman. For this reason, keys like `Label` may be set multiple times while for others that may be set only once, such as `Entry`, only the last occurrence is used. Furthermore, using only the last occurrence allows overriding the value in drop-in unit files.

As for you case, have you tried using the line continuation feature? For the example above it will look like:
```
[Container]
Image=docker.io/library/traefik
Exec=--api.insecure=true \
--providers.docker=true \
--providers.docker.exposedbydefault=false \
--entryPoints.web.address=:80
Notify=true
Volume=%t/podman/podman.sock:/var/run/docker.sock
SecurityLabelDisable=true
```

**Reply by dmotte** [reply]
Thanks! The line continuation feature worked for me :slightly_smiling_face: using **Podman 5.4.2** on Debian 13 (trixie)
